distribution/std-lib/Base/src/Meta/Meta.enso

```diff
@@ -0,0 +1,83 @@
+from Base import all
```

Comment:
This file needs a big doc stating that it exposes _some_ implementation details, and hence APIs may be subject to change.